driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -291,7 +268,7 @@ pub mod tests {
         });
 
         let expected_solution = models::Solution {
-            objective_value: U256::from_dec_str("15854632034944469292777429010439194350").ok(),
+            objective_value: None,
```

Comment:
Would it make sense to take the objective value out of the solution struct completely, or do we use it somewhere?

Reply:
It is also used in the whole snapp infrastructure. I don't want to touch this in this PR.